### DISCUSSION TITLE
DataCap Refresh Not Approved - BigData Exchange (BDX)

### DISCUSSION THREAD

**Discussion by Kevin-FF-USA** [filecoin-project/Allocator-Governance#246] [General]
**Organization**
[1027](https://github.com/filecoin-project/Allocator-Registry/blob/main/Allocators/1027.json) | BigData Exchange, BDX Fil+ Allocator | Manual | [Bookkeeping](https://github.com/cryptoAmandaL/BDE-Allocator) | Singapore | https://github.com/filecoin-project/notary-governance/issues/1027 | [f03018484](https://filecoinpulse.pages.dev/allocator/f03018484/#pageparamsallocator_id)

**Past Refresh History**
May 22, 2024: [Refresh 1 = Throttled at 2.5PiB](https://github.com/filecoin-project/Allocator-Governance/issues/20)
Notes from Governance: 
- In addition to updating this team to the correct pathway type, this allocator is demonstrating mixed compliance. 
- Mixed evidence of diligence with clients (no verification of client claims)
- Subsequent allocations given despite noncompliant client deal-making, with minimal allocator intervention through comments
- No retrievability for datasets, despite claims of public open data by both allocator and client (not showing distributed network data storage utility)
- Given this evidence, we will need to update the documentation to reflect the reality of this manual pathway. If this team is in fact building a market-based platform that utilizes smart contracts for data distribution, we would love to see that! We encourage them to reapply for that pathway under the new rolling application structure.

August 20, 2024: [Refresh 2 = Throttled at 2.5PiB](https://github.com/filecoin-project/Allocator-Governance/issues/142)
Notes from Governance: 
- Low and inconsistent retrieval of public open datasets, average retrievability success rate of 17.55%
- High usage of VPNs without enforcing stated requirements from allocator application  "require clients to disclose their VPN usage" 
- Also claimed to utilize "monitoring system to track and report VPN usage", but no evidence.
- Noncompliant SP distribution (Data Distribution section of application), with continued subsequent allocations 
Noncompliant DataCap tranche allocations.
- Mixed KYC diligence into client claims, such as clients making conflicting region claims and lacking KYC checks in GitHub 
- As a reminder, the allocator team is responsible for verifying, supporting, and intervening with their clients. If a client is NOT providing accurate deal-making info (such as incomplete or inaccurate SP details) or making deals with noncompliant unretrievable SPs, then the allocator needs to intervene and require client updates before more DataCap should be awarded.

October 14, 2024: [Refresh 3 = Final Throttle at 2.5PiB](https://github.com/filecoin-project/Allocator-Governance/issues/191)
Notes from Governance:
- **Without the following areas being corrected, this allocator will be rejected**
- Retrieval scores increasing above 75%
- Dataset replicas being accurately reported and investigated
- SP distribution lists remaining accurate, with justification
- VPN usage being noted and tracked
- Regional distribution increasing to match application requirements**
- Overall, the clients working with this pathway are not showing significant improvement, and this allocator is not showing evidence of developing or delivering on its initial claims of building market-based, smart contract data onboarding tooling. We will request a final 2.5PiB of DataCap, in the hopes that we can see any progress towards the design goals presented initially.

November 25, 2024 [Refresh 4 = Rejected](https://github.com/filecoin-project/Allocator-Governance/issues/235)
Notes from Governance:
- Noncompliant and inaccurate regional distribution, with VPNs unaccounted for
- No additional details regarding dataset preparation, especially with these redundant datasets
- Overall retrieval still below 25%
- These flags have been brought up multiple times, and while we understand there can be some delays in improvement, we have quite a few successful manual allocators. Additionally, there are new teams that are eager to join the program and have already shown success collaborating with allocators. **At this point, the governance team is going to intervene and reject this application. We are requesting the RKH to remove this pathway.**

**Discussion** 
Big Data Exchange was not recommended for a refresh due to not addressing issues flagged over the course of (4) throttled DataCap Refresh applications.

It's important to ensure that the decision-making process remains efficient, effective, scalable, and open. If BDX or a member of the community would like to present information or data as to why this pathway was in fact operating within the strucutre of their own applications standards and should receive a datacap refresh, this is the discussion forum to do so.
**Public forum for BDX, Community, Governance to discuss the non-renewal decision.**

**Comment by cryptoAmandaL** [comment]
>Thanks for your comment. Our original goal was to use automated tools to form an innovative distribution pathway. However, due to difficulty of development and the actual situation, we decided to respond to clients' applications with priority, manually allocating datacap to them. We are following the rules as an allocator.

On the first round of reviews we've already explained why we're choosing manual way.

What we want to say is mostly in this reply.
https://github.com/filecoin-project/Allocator-Governance/issues/235#issuecomment-2522907068

We do not agree with this result. 
We ask to keep our eligibility as allocator.
We do not accept the unfair treatment, which should not be allowed in the fil+ community.


1. We think there's some wrong in your conclusion.

>Noncompliant and inaccurate regional distribution, with VPNs unaccounted for

We have communicated about this problem in our [second review](https://github.com/filecoin-project/Allocator-Governance/issues/142).
Here's the record.
https://github.com/filecoin-project/Allocator-Governance/issues/142#issuecomment-2340043273
![image](https://github.com/user-attachments/assets/13a2da0d-5066-44ab-9f0b-ae2e7c957ace)
https://github.com/filecoin-project/Allocator-Governance/issues/142#issuecomment-2346859273
![image](https://github.com/user-attachments/assets/224fa12e-aee6-478a-9240-d5df8b374f09)
https://github.com/filecoin-project/Allocator-Governance/issues/142#issuecomment-2348504892
![image](https://github.com/user-attachments/assets/5985b165-61e7-41f7-bbf6-b5b76e0cd3fe)


We are always looking for good tools to help us or other allocators who are having the same require recognize sp's attribute.
We believe that each allocator's standard can be considered accurate until there is a standardization tool to provide a standard answer.



>No additional details regarding dataset preparation, especially with these redundant datasets

We do not believe that public academic datasets are superior or inferior. That's why we welcome all kinds of data from legitimate sources from our clients. 

Nowadays, all alloctors are allowing redundant datasets. The value of data should not be denied based on the number of times it has been stored.

Can the community give us some ideas? 

If allocator can't follow its own criteria for accepting data, what's the best way to accept clients?



>Overall retrieval still below 25%

Can you please tell us how to get the result?

Here's the latest report. From previous report and the latest report, the overall retrieval rates are both higher than 25%.
https://check.allocator.tech/report/cryptoAmandaL/BDE-Allocator/issues/26/1733472599300.md
![image](https://github.com/user-attachments/assets/b5fa90ca-5434-4be1-8367-e1484d1cab8d)

https://check.allocator.tech/report/cryptoAmandaL/BDE-Allocator/issues/28/1733472604920.md
![image](https://github.com/user-attachments/assets/60e26253-7b87-47c4-ad66-e9694b84da10)


When sps meet problems, they need time to resolve that.

Also, the data in report is hysteretic. We usually get accurate results from spark dashboard.

This sp in report has got only 9% of successful retrieval rate. The result I got from spark is 86% one minute ago.
![image](https://github.com/user-attachments/assets/007d04a9-1f43-4afe-978a-e10b8ed67ffb)


2. There are a number of allocators claimed their type is market-based, and no successful one has been seen yet.

Allocator is a totally new experiment for the Fil+ community. Every one is exploring and improving on how to be better as an allocator. Why is it only unfair to us?



3. If your conclusion is not corrected this time, we'll keep feeding and complaining in community.

We think that it is a web3 community here.

Decision-making can be voted by community members, rather than individual preferences governing everything. This is not a stage for dictators.


Expect to receive a response.

**Comment by cryptoAmandaL** [comment]
@galen-mcandrew @Kevin-FF-USA Hello, what's the next step here?

**Comment by Kevin-FF-USA** [comment]
Thanks @cryptoAmandaL,
Appreciate the threading out here for discussion.

Maybe one of the first places to come back to is the affiliation topic.
https://github.com/filecoin-project/Allocator-Governance/issues/235#issuecomment-2539502338

In the [accepted application for this pathway to become an Allocator](https://github.com/filecoin-project/notary-governance/issues/1027), this pathway was selected as an operational entity of [bigdataexchange.io](https://www.bigdataexchange.io/).

For community review & transparency, recommend sharing two things.
1. Developmental roadmap or repo for the Market Based pathway that has been in development?
2. Your affiliation with BDX.  What is the status of the org?  
<img width="897" alt="image" src="https://github.com/user-attachments/assets/8add5e53-9686-48ea-af4f-f59f5e6ce905" />

**Comment by cryptoAmandaL** [comment]
@Kevin-FF-USA 
https://github.com/cryptoAmandaL/BDXA
The code we have uploaded into this repo. This is unfinished code because we don't have enough technology or money to finish it at the moment. The decision to apply for the market-base allocator was based on the fact that we had successfully launched a market-based platform for trading data. As our previous item has sunset, we turned to build a new allocator pathway. In actual practice, we've used our experience with the fil+ community to consider our current situation to be more suited to a manual allocator. 
I'm one of BDX. Some members from BDX are still working together to be deeply participating in the filecoin community and contributing to fil+ building.

**Comment by Kevin-FF-USA** [comment]
Hi @cryptoAmandaL,
I'm not authorized to make the decisions to keep this pathway in the program, but can help by keeping this issue open and give a few recommendations. 

Would suggest
1. Having a current member of the BDX leadership (company) comment in this issue.  Give their pledge that they are aware of this pathway and fully support its standards being tied to their reputation.

2. Pausing on the Manual pathway and investing in development on the Market platform.  
*If you have any clients, direct them to a pathway of similar standards.  Here is a list [Allocators])(https://fil.org/filecoin-plus/allocators)

  If un-able to find a pathway for your client, please reach out to me with an @ in the Fil-Plus-Allocators channel and I'll commit to personally working with you to find a match with other Allocators. 

Realize this may not be the solution that you'd like to see, but this is what is needed to ensure that pathways are onboarding data for the benefit of the program overall.  

As always, here for any questions you may have.
-Kevin

**Comment by cryptoAmandaL** [comment]
Hello @Kevin-FF-USA ,
I need a reasonable reply before gov team ask us to do anything further. I have already responded earlier to the questions you have raised in my [issue](https://github.com/filecoin-project/Allocator-Governance/issues/235), but I don't think you have resolved the questions I have raised.


1. Why did the gov team ignore the part of the clients' retrieval enhancements?

You keep insisting that the client's retrieval result is not up to standard even after I have given the client's latest retrieval result.
>Here's the latest report. From previous report and the latest report, the overall retrieval rates are both higher than 25%.
https://check.allocator.tech/report/cryptoAmandaL/BDE-Allocator/issues/26/1733472599300.md

>https://check.allocator.tech/report/cryptoAmandaL/BDE-Allocator/issues/28/1733472604920.md


>When sps meet problems, they need time to resolve that.

>Also, the data in report is hysteretic. We usually get accurate results from spark dashboard.

>This sp in report has got only 9% of successful retrieval rate. The result I got from spark is 86% one minute ago.
![image](https://github.com/user-attachments/assets/7b95e480-ea1c-42f7-8d45-14ba6fffefb3)



2. Why not treat every allocator fairly?

There are allocators that do not follow the rules and are still continued to be granted datacap, such as NonEntropy.
https://github.com/filecoin-project/Allocator-Governance/issues/235#issuecomment-2527524244
We had questioned it and then went to their application and questioned it again as requested by the gov team. But gov team is not paying attention to my doubts. 
![image](https://github.com/user-attachments/assets/a48dea76-0a34-4977-afdb-0e7d287733ce)

![image](https://github.com/user-attachments/assets/9146f537-30a4-4db1-b9c1-d64a0e116ce5)


Exactly which way should I raise my doubt in order to be dealt with?



**I hope that the unfairness I have suffered will be explained first.**

**Comment by cryptoAmandaL** [comment]
@Kevin-FF-USA Hello, are there any reply? We want to solve the matter.

**Comment by cryptoAmandaL** [comment]
@Kevin-FF-USA Will I get any update here? Or should I go to slack or email?

**Comment by Kevin-FF-USA** [comment]
Hi @cryptoAmandaL 

Believe this is pathway was removed for both the lack of standards maintained (detailed in top post), but more recently - for the [potential of misrepresentation](https://github.com/filecoin-project/Allocator-Governance/issues/235#issuecomment-2539502338) for organizational affiliation.

Have you been in contact with the leadership team at BDX and have a member who is willing to comment in this issue?
https://github.com/filecoin-project/Allocator-Governance/discussions/246#discussioncomment-11575750

Warmly,
-Kevin
